site/content/reference/cli-help.md

```diff
@@ -39,11 +39,12 @@ This command configures NGINX for a single NGINX Gateway Fabric resource.
-| _usage-report-server-url_    | _string_ | The base server URL of the NGINX Plus usage reporting server. |
-| _usage-report-cluster-name_  | _string_ | The display name of the Kubernetes cluster in the NGINX Plus usage reporting server. |
-| _usage-report-skip-verify_   | _bool_   | Disable client verification of the NGINX Plus usage reporting server certificate. |
+| _product-telemetry-disable_         | _bool_   | Disable the collection of product telemetry (Default: `false`).                                                                                                                                                                                                                                                                                                                          |
+| _usage-report-secret_               | _string_ | The namespace/name of the Secret containing the credentials for NGINX Plus usage reporting.                                                                                                                                                                                                                                                                                              |
+| _usage-report-server-url_           | _string_ | The base server URL of the NGINX Plus usage reporting server.                                                                                                                                                                                                                                                                                                                            |
+| _usage-report-cluster-name_         | _string_ | The display name of the Kubernetes cluster in the NGINX Plus usage reporting server.                                                                                                                                                                                                                                                                                                     |
+| _usage-report-skip-verify_          | _bool_   | Disable client verification of the NGINX Plus usage reporting server certificate.                                                                                                                                                                                                                                                                                                        |
+| _snippets-filters_           | _bool_   | Enable SnippetsFilters feature. SnippetsFilters allow inserting NGINX configuration into the generated NGINX config for HTTPRoute and GRPCRoute resources.                                                                                                                                                                                                                               |
```

Comment:
Is this supposed to be offset with the lines around bool and the end? When I view the file in entirety, it shows the end `|` is not aligned?

Reply:
fixed ( I think)

Reply:
lgtm!